lib/_http_outgoing.js

```diff
@@ -849,6 +871,11 @@ OutgoingMessage.prototype.end = function end(chunk, encoding, callback) {
       this.socket.cork();
     }
 
+    const byteCount = (this[kBytesWritten] + chunk.length);
```

Comment:
This may not be entirely accurate in every case due to content encoding. For instance, if I pass in `w.end('afef', 'hex')`, I'm only writing 2 bytes. The calculation here needs to take into consideration the `encoding` and the actual byte length.